internal/cortex/querier/queryrange/queryrange.proto

```diff
@@ -98,18 +98,39 @@ message PrometheusResponseQueryableSamplesStatsPerStep {
 message SampleStream {
   repeated cortexpb.LabelPair labels = 1 [(gogoproto.nullable) = false, (gogoproto.jsontag) = "metric", (gogoproto.customtype) = "github.com/thanos-io/thanos/internal/cortex/cortexpb.LabelAdapter"];
   repeated cortexpb.Sample samples = 2 [(gogoproto.nullable) = false, (gogoproto.jsontag) = "values"];
+  repeated SampleHistogramPair histograms = 3 [(gogoproto.nullable) = false, (gogoproto.jsontag) = "histogram"];
 }
 
 message Sample {
   repeated cortexpb.LabelPair labels = 1 [(gogoproto.nullable) = false, (gogoproto.jsontag) = "metric", (gogoproto.customtype) = "github.com/thanos-io/thanos/internal/cortex/cortexpb.LabelAdapter"];
-  cortexpb.Sample sample = 2 [(gogoproto.nullable) = false, (gogoproto.jsontag) = "value"];
+  double sampleValue = 2 [(gogoproto.jsontag) = "value"];
```

Comment:
I understand that this is a breaking change and invalidating cache is necessary. I made the change to be closer to the prometheus common model.

Reply:
I think this is fine as long as it's reflected in the CHANGELOG.